audio/BoCA/Portfile

```diff
@@ -5,11 +5,15 @@ PortGroup           github 1.0
 PortGroup           makefile 1.0
 PortGroup           compiler_blacklist_versions 1.0
 
-github.setup        enzo1982 BoCA 1.0.5 v
-revision            0
-checksums           rmd160  5665880a576d3f90e7754362be6bfd1b414b3c4d \
-                    sha256  9befa4555b5d9a0ecf3b8472f1d8f14c26dee50993d1d66f35a2d0f140a72870 \
-                    size    2634890
+# Use commit for now due to several fixes from upstream, see:
+# https://github.com/enzo1982/BoCA/issues/12
+# https://github.com/enzo1982/BoCA/pull/13
+github.setup        enzo1982 BoCA 4ea79d518b508d43e16a0b86a313e99342efcc71
+version             1.0.7
+revision            1
```

Comment:
If the `version` increases, then `revision` should return to zero.

Reply:
Missed that. Will fix now. Thanks.

Reply:
@herbygillot CI passed, all good.